### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by jesuspg**
#### Reviewers
@hmunfru @jframos 

#### Description
solution: check accept header value.

#### Testing
Locally.

### CODE REVIEW

File: rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/auth/OpenStackAuthenticationFilter.java [hunk 1]
```diff
@@ -158,12 +171,12 @@ public final void doFilter(final ServletRequest req, final ServletResponse res,
...
+            if (headerToken == null) {
+                headerToken = "";
             }
 
             try {
-                String token = header;
+                String token = headerToken;
                 if ("".equals(token)) {
                     String str = "Missing token header";
                     logger.info(str);
```
Comment (flopezag) [inline rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/auth/OpenStackAuthenticationFilter.java]: Why logger.info and not logger.error?

**Comment by flopezag** [comment]
Could you give more details about the problem in the description of the PR?

File: rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/auth/OpenStackAuthenticationFilter.java (hunk 1 above)
Comment (jesuspg) [inline rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/auth/OpenStackAuthenticationFilter.java]: always was logger.info!

**Comment by jesuspg** [comment]
@flopezag CLAUDIA-4505 in private jira: When I execute a HTTP request with invalid Accept header, *HTTP 200* is recived instead of *HTTP 406*

File: rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/auth/OpenStackAuthenticationFilter.java (hunk 1 above)
Comment (flopezag) [inline rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/auth/OpenStackAuthenticationFilter.java]: LGTM

**Comment by flopezag** [comment]
LGTM

**Pull request merged**